pages/common/xmlstarlet.md

```diff
@@ -0,0 +1,33 @@
+# xmlstarlet
+
+> A commandline XML/XSLT toolkit.
+> Note: You will likely need to know XPath: <https://developer.mozilla.org/en-US/docs/Web/XPath>.
+> More information: <https://xmlstar.sourceforge.net/docs.php>.
+
+- Format an XML document and print to stdout:
+
+`xmlstarlet format {{path/to/file.xml}}`
+
+- XML document can also be piped from stdin:
+
+`{{cat path/to/file.xml}} | xmlstarlet format`
+
+- Print all nodes that match a given XPath:
+
+`xmlstarlet select --template --copy-of {{xpath}} {{file.xml}}`
```

Comment:
```suggestion
`xmlstarlet select --template --copy-of {{xpath}} {{path/to/file.xml}}`
```

Reply:
Is it really necessary to repeat this over and over again when it's just an input file? I feel like it's just increasing the line length without a real benefit.

If this is a style you wish to globally enforce, I think it would be best to formalise it in CONTRIBUTING.md.

Reply:
That is true, we should formalize it.